docs/docs/how-tos/domain-registry.md

```diff
@@ -20,7 +20,12 @@ During deployment, Nebari will generate an [Ingress](https://kubernetes.io/docs/
 Take IP Address 12.312.312.312 and update DNS to point to "your.domain" [Press Enter when Complete]
 ```
 
-Once the IP is generated, you will need to grab it and create the necessary records within the DNS provider of your choice.
+Once the IP is generated, you will need to grab it and create the necessary records within the DNS provider of your choice. Setting a DNS record heavily depends on your provider, so an internet search for A/CNAME record for your specific provider should yield helpful results.
+
+:::note
+During the initial deployment, Digital Ocean, GCP, and Azure will display an "IP" address, that you can use to set the A record.
+Whereas, AWS will display a "hostname" that you can use to set the CNAME record.
+:::
```

Comment:
@iameskild This is what I gathered from [our old docs](https://docs.qhub.dev/en/stable/source/installation/setup.html#domain-registry), is it accurate?

Reply:
Yep, this looks accurate to me :)